library/nist_kw.c

```diff
@@ -57,6 +57,20 @@
 #define KW_SEMIBLOCK_LENGTH    8
 #define MIN_SEMIBLOCKS_COUNT   3
 
+/* constant-time buffer comparison */
+static inline int mbedtls_nist_kw_safer_memcmp( const void *a, const void *b, size_t n )
```

Comment:
Now that we have the `platform_util` module, we should use it instead of duplicating code. Please move all the definitions of `xxx_safer_memcmp` to `platform_util` and, since that makes the function a public API, please add a few test cases to check that the function is functionally correct.

Reply:
This is a differnt feature, and extends the scope of this PR. It should be done in a different PR